spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
```

Comment:
This part about the tension was unclear (perhaps jumping into details too quickly). Perhaps the part on the alternative can be moved into a separate info box with more details - I think many readers may not get the reference to "more sophisticated modelling".

Reply:
Fair point. I think it is important to motivate the addition of triple terms though, which is more advanced than simple statements. A motivation and strength of reified triples is to provide this bridge between granularities. I'll think more about how to present it.

Reply:
A middle ground could be to extract the sentence about the trade-off in an `<aside class=note>`.

Reply:
I realize that @william-vw made the same suggestion [below](https://github.com/w3c/rdf-primer/pull/16/files#r1876149102)

Reply:
Yes. I'm doing some reordering, following @pchampin's [suggestion](https://github.com/w3c/rdf-primer/pull/16#discussion_r1880566872). I'll resolve this part here (to help my own sense of what I've addressed).